### DISCUSSION TITLE
Add configs of PPC/x86 Mac OS X

### DISCUSSION THREAD

**Discussion by KendrickLamar2003** [utmapp/UTM#3679] [UTM Help (macOS)] [answered]
For a long time, I have been scouring the internet for help with installing 10.5/6 on UTM, but to no avail. Could you provide a config for PPC/x86 Mac OS X in the gallery?

**Comment by adespoton** [comment, marked as answer]
Feel free to use mine: https://github.com/adespoton/utmconfigs

I don't provide the install images, but the notes provide instructions on how to get them.

Unfortunately, i386 versions of 10.4, 10.5 and 10.6 are still missing, because they're still broken.  The required OpenCore configuration that's used for x86-64 10.7+ doesn't work with them, and the Dortania/Acidanthera/Khronokernel configs don't boot correctly either.  It's not just a UTM thing; while I can get them working in KVM/X86, I haven't got them working on QEMU/M1 at all yet.

If anyone does know what components are misconfigured, please let me know, and I'll get the configs fixed so they can go to the gallery.

**Comment by adespoton** [comment]
One other note: for the older PPC images (10.0 and 10.1), you'll want to capture the mouse prior to the UI coming up and leave it captured, as SPICE clobbers I/O support if you try to bind it later than that (SPICE uses USB, and the config needed for 10.0 and 10.1 doesn't understand USB, but uses ADB instead).
